src/statmech.jl

```diff
@@ -175,17 +175,10 @@ function chemical_equilibrium_equations(T, nₜ, nₑ, absolute_abundances, ioni
     end
 end
 
-###################################################################################
-# The methods below are experimental, and not used by Korg for spectral synthesis #
-###################################################################################
 
 """
     hummer_mihalas_w(T, n, E_level, nH, nHe, ne)
```

Comment:
I think that sounds like a sensible plan! 

More generally, it's a little unclear to me if it would ever stop being a "derived work" even if you rewrite it (since your from-scratch implementation might still be influenced by the earlier work), but in this context I definitely don't think that would be an issue.

Reply:
done.